### PR TITLE
Update ToC

### PR DESCRIPTION
**Pull request by hwwhww**

### CODE REVIEW

File: specs/core/0_beacon-chain.md [hunk 1]
```diff
@@ -65,7 +65,7 @@
             - [`is_slashable_validator`](#is_slashable_validator)
             - [`is_slashable_attestation_data`](#is_slashable_attestation_data)
             - [`is_valid_merkle_branch`](#is_valid_merkle_branch)
-        - [Misc](#misc)
+        - [Misc](#misc-1)
```
Comment (CarlBeek) [inline specs/core/0_beacon-chain.md]: I like this resolution of duplicate headers. Can you do `#### Helper functions` and `#### Rewards and penalties` too please?

Comment (CarlBeek) [reply specs/core/0_beacon-chain.md]: Also `### Attestations` in the V-guide.

Comment (hwwhww) [reply specs/core/0_beacon-chain.md]: I think they are already in this format, aren't they? :)

Comment (CarlBeek) [reply specs/core/0_beacon-chain.md]: Oh, they are. My bad. 🤦‍♂

**Pull request merged**